extensions/src/main/java/org/pentaho/platform/plugin/services/security/userrole/oauth/IPentahoOAuthHandler.java

```diff
@@ -0,0 +1,31 @@
+/*! ******************************************************************************
+ *
+ * Pentaho
+ *
+ * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ *
+ * Use of this software is governed by the Business Source License included
+ * in the LICENSE.TXT file.
+ *
+ * Change Date: 2029-07-20
+ ******************************************************************************/
+
+package org.pentaho.platform.plugin.services.security.userrole.oauth;
+
+import org.pentaho.platform.api.mt.ITenant;
+import org.pentaho.platform.security.userroledao.PentahoOAuthUser;
+
+import java.util.List;
+
+public interface IPentahoOAuthHandler {
+
+  boolean isUserAccountEnabled( String registrationId, String clientCredentialsToken, String userId, boolean retry );
```

Comment:
Add javadocs to class and interface functions for IPentahoOAuthHandler.java

Reply:
addressed